packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -106,6 +127,14 @@ class ScrollBehavior {
       case TargetPlatform.windows:
         return child;
       case TargetPlatform.android:
```

Comment:
Seeing lots of platform checks, would one be able to use/enable this feature in non-android platforms? I mean enabling the feature globally!

Reply:
Yes, It is possible to extend `ScrollBehavior` and override `buildOverscrolIIndicator` for your own implementation.

We recently made it possible to set the scroll behavior for your whole app with `[Widgets, Material, Cupertino]App.scrollBehavior`. Alternatively, The `ScrollBehavior`, inherited through a `ScrollConfiguration`, can be added anywhere in your app to target specific widgets and subtrees. :)

Reply:
Thanks